docs/manual/tags.md

```diff
@@ -344,259 +182,189 @@ denotes the scriptlet interpreter and possible arguments, and flags tag
-
-  Source file names.
-
-* Sourcepackage (1106) int32
-
-  Denotes a source rpm.
+Buildarchs    | 1089 | string array | If present, specifies the architectures the package may be built for.
+Excludearch   | 1059 | string array | If present, limits the architectures on which the package is buildable by excluding those specified.
+Excludeos     | 1060 | string array | If present, limits the operating systems on which the package is buildable by excluding those specified.
+Exclusivearch | 1061 | string array | If present, limits the architectures on which the package is buildable exclusively to those specified.
+Exclusiveos   | 1062 | string array | If present, limits the operating systems on which the package is buildable exclusively to those specified.
+Nopatch       | 1052 | int32 array  | Denotes a patch number for which source is not supplied.
+Nosource      | 1051 | int32 array  | Denotes a source number for which source is not supplied.
+Patch         | 1019 | string array | Patch file names.
+Source        | 1018 | string array | Source file names.
+Sourcepackage | 1106 | int32        | Denotes a source rpm.
 
 ## Internal / special
 
-* Headeri18ntable (100) string array
-
-  Locales for which the header has translations.
-
-* Headerimmutable (63) bin
-
-  Special tag to return the unmodified, original image of the header
-  even after data has been added to it in eg installation.
+Headeri18ntable | 100 | string array | Locales for which the header has translations.
+Headerimmutable | 63  | bin          | Special tag to return the unmodified, original image of the header even after data has been added to it in eg installation.
 
 ## Deprecated / Obsolete
-* Filecontexts (1147) string array
-* Fscontexts (1148) string array
-* Gif (1012) bin
-* Icon (1043) bin
-* Oldenhancesname (1159) string array
-* Oldenhancesversion (1160) string array
-* Oldenhancesflags (1161) int32 array
-* Oldfilenames (1027) string array
-* Oldsuggestsname (1156) string array
-* Oldsuggestsversion (1157) string array
-* Oldsuggestsflags (1158) int32 array
-* Patchesflags (1134) int32 array
-* Patchesname (1133) string array
-* Patchesversion (1135) string array
-* Pubkeys (266) string array
-* Recontexts (1149) string array
-* Removetid (1129) int32
-* Xpm (1013) bin
+
+These tags are not longer in active use. If encountered in packages
+they are ignored.
+
+Filecontexts       | 1147 | string array
+Fscontexts         | 1148 | string array
+Gif                | 1012 | bin
+Icon               | 1043 | bin
+Oldenhancesname    | 1159 | string array
+Oldenhancesversion | 1160 | string array
+Oldenhancesflags   | 1161 | int32 array
+Oldfilenames       | 1027 | string array
+Oldsuggestsname    | 1156 | string array
+Oldsuggestsversion | 1157 | string array
+Oldsuggestsflags   | 1158 | int32 array
+Patchesflags       | 1134 | int32 array
+Patchesname        | 1133 | string array
+Patchesversion     | 1135 | string array
+Pubkeys            | 266  | string array
```

Comment:
This tag is actually used by RPM internally.  It is ignored if found in the signature header of a package, but if it is found in the main header of a package, it adds a key to the RPM database.

Reply:
That it is actually used by rpm is true (my bad in the original doc), but the rest is not for any remotely relevant version (rpm 4.4.x behaved somewhat like that). When you don't know for sure, at the very least add an "I think" disclaimer or such, misinformation is bad for everybody.

It's not looked for or used from package files at all, and certainly doesn't "add a key" to the rpmdb. Rpm only ever looks for RPMTAG_PUBKEYS in gpg-pubkey "packages" in the rpmdb.

Reply:
Sorry, I should have said, “when found in a gpg-pubkey package”.

Reply:
Moved to the Internal / Special section.